go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -268,6 +266,27 @@ func (c *Controller) syncServices(key string) error {
 			"Error listing Endpoint Slices for Service %s/%s: %v", namespace, name, err)
 		return err
 	}
+
+	// If idling enabled and there are no endpoints, we need to move the VIP from the main loadbalancer
+	// ,that has the reject option for backends without endpoints, to the idling loadbalancer, that
+	// generates an event to kick of the unidling process in the openshift controller manager.
+	// if not, we delete the vips from the idling lb and move them to their right place.
+
+	vips := c.collectServiceVIPs(service)
+	if svcNeedsIdling(service.Annotations) && len(endpointSlices) == 0 {
+		c.addServiceToIdlingBalancer(vips, service)
+		vipsTracked = vipsTracked.Delete(vips.List()...)
```

Comment:
if you return , what are you going to do with the remaining tracked vips?
those has to be deleted, it means that the service has mutated and those ipvs no longer exist

Reply:
Good catch! This is a leftover from an intermediate implementation where the st was used in both deleteVIPsFrom functions. The argument is still there, but not used anymore :-)

Reply:
Yes